develop-seat-adjuster.md

```diff
@@ -131,31 +131,40 @@ But it still makes sense to get a general idea of how to test and debug the code
 
 As mentioned before, the execution and, thus, the testing of the application requires a set of other components to be available
 like the KUKSA Databroker or an MQTT-broker.
-Furthermore, the deployment, the configuration, and the behavior of the application may change depending on the used (container) management solution.
+Furthermore, the deployment, the configuration, and the behavior of the application may change depending on the used container management solution.
 Therefore, Eclipse Velocitas allows the deployment of the required components and the application with the container management approach of choice inside the DevContainer.
 
-The different runtimes are maintained in [a separate Eclipse Velocitas repository](https://github.com/eclipse-velocitas/devenv-runtimes). To start a runtime,
-we use the [Eclipse Velocitas CLI](https://github.com/eclipse-velocitas/cli):
+The different runtimes are maintained in [a separate Eclipse Velocitas repository](https://github.com/eclipse-velocitas/devenv-runtimes). To start a runtime.
```

Comment:
shouldn't "To start a runtime" be removed as you add it to the next sentence?

Reply:
True, good finding.

Reply:
I update the respective sentence.